java/src/main/java/genericpubsub/ManagedSubscribe.java

```diff
@@ -195,25 +209,15 @@ private Schema getSchema(String schemaId) {
         });
     }
 
-    /**
-     * Helper function to decode the payload from the stream.
-     */
-    public static GenericRecord deserialize(Schema schema, ByteString payload) throws IOException {
```

Comment:
We already have a `deserialize` function in [CommonContext](https://github.com/forcedotcom/pub-sub-api/blob/main/java/src/main/java/utility/CommonContext.java#L259-L264) so I'm proposing we remove this here and just use the CommonContext function